src/Modules/Guilds/pages/LandingPage/LandingPage.tsx

```diff
@@ -82,7 +78,7 @@ const LandingPage: React.FC = () => {
           {' '}
           <StyledLink to={`createGuild`}>{t('guilds.create')}</StyledLink>
         </StyledButton>
-      </InputContainer>
+      </InputContainer> */}

```

Comment:
Should we delete this instead of commenting it?

Reply:
I think its fine to keep it in for now as the design will stay the same we are just removing it since we don't currently have the features of search and a new guild page implemented yet

Reply:
Took a second look and realised this isn't doing much since the code was moved to styles so removed the comments